contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -325,6 +341,20 @@ contract FunctionsRouter is RouterBase, IFunctionsRouter, FunctionsSubscriptions
     result = CallbackResult(success, gasUsed);
   }
 
+  function isValidCallbackGasLimit(uint64 subscriptionId, uint32 callbackGasLimit) external view {
```

Comment:
Why do we have 2 functions?  Can we simplify by only having a single isValidCallbackGasLimit function and make that function `external view`?

Reply:
For gas savings. Internal functions are cheaper to call.

Reply:
I guess I assumed the optimizer would handle things like this, but maybe not.